src/style.css

```diff
@@ -0,0 +1,97 @@
+:root {
```

Comment:
There should not be a src/style.css file - is this a mistake?

Reply:
ah, leftover from the initial Vite starter